upi/vsphere/network/output.tf

```diff
@@ -0,0 +1,11 @@
+output "master_ips" {
+  value = "${split(" ", trimspace(data.external.ping.result.master_ips))}"
+}
+
+output "worker_ips" {
+  value = "${split(" ", trimspace(data.external.ping.result.worker_ips))}"
+}
+
+output "bootstrap_ip" {
+  value = "${split(" ", trimspace(data.external.ping.result.bootstrap_ip))}"
```

Comment:
I would prefer to keep this as a string instead of a list, unless it is terribly difficult otherwise. Would it work to just omit the `split`?